docs/docsite/rst/network/user_guide/cli_parsing.rst

```diff
@@ -0,0 +1,728 @@
+.. _cli_parsing:
+
+*****************************************
+Parsing Semi-structured Text with Ansible
+*****************************************
+
+The `ansible.netcommon <https://galaxy.ansible.com/ansible/netcommon>`_ collection version 1.2.0 or later  includes the :ref:`cli_parse <ansible_collections.ansible.netcommon.cli_parse_module>` module that can run commands and parse the output. You can use the ``cli_parse`` module on a device, host, or platform that only supports a command-line interface and the commands issued return semi-structured text.
+
+.. contents::
+   :local:
+
+
+Understanding the CLI parser
+=============================
+
+The ``cli_parse`` module can either run a CLI command on a device and return a parsed result or can simply parse any text document. The ``cli_parse`` module includes cli_parser plugins to interface with a variety of parsing engines.
+
+Why parse the text?
```

Comment:
I think I might put this paragraph (the one below) at the top of the page as an intro, above the local TOC. It tells people why they want this feature.

Reply:
put part of it up there instead of the current text to keep the before toc text short but helpful  (and improve the google description, which should pull that first sentence or two in search results)